pinot-common/src/main/java/com/linkedin/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -52,32 +56,21 @@
   private SelectionResults _selectionResults;
   private List<AggregationResult> _aggregationResults;
 
-  private Map<String, String> _traceInfo;
-  private List<QueryProcessingException> _processingExceptions;
-  private List<String> _segmentStatistics;
-
-  public static final BrokerResponseNative EMPTY_RESULT;
-  public static final BrokerResponseNative NO_TABLE_RESULT;
-
-  static {
-    EMPTY_RESULT = new BrokerResponseNative();
-    EMPTY_RESULT.setTimeUsedMs(0);
+  private Map<String, String> _traceInfo = new HashMap<>();
+  private List<QueryProcessingException> _processingExceptions = new ArrayList<>();
```

Comment:
why create arraylist/hashmap every time when we don't have exceptions most of the time.